db/prompts.go

```diff
@@ -144,11 +150,22 @@ func GetAllPrompts(filters SupabaseFilters) ([]Prompt, error) {
 			value = "{" + value + "}"
 		}
 
+		if filter.Operation == Ilike || filter.Operation == Like {
+			value = "%" + value + "%"
```

Comment:
Not a big issue for this specific case but this is a bad practice that can lead to injections (in this case the injection just allows the user to send wildcards)